scraibe/transcript_exporter.py

```diff
@@ -62,6 +167,18 @@ def annotate(self, *args, **kwargs) -> dict:
         
         return self
     
+    def _remove_hallucinations(self) -> None:
+        segments_to_drop=[]
+
+        for id in self.transcript:
+            for language, snippets in WHISPER_HALLUCINATIONS.items():
```

Comment:
Maybe just using a list of hallucinations instead of dict (the transcript does not know which language the transcript has)